src/tetromiq.py

```diff
@@ -14,7 +14,7 @@ def game():
     paused = False
     game_over = False
     fall_speed = INITIAL_FALL_SPEED
-    level = 1
+    previous_level = 1
```

Comment:
Why not only `level =1`?

Reply:
I use previous_level to determine when the level change, so the previous_level is a better name